Wikipedia/Code/WMFFontSliderViewController.xib

```diff
@@ -35,16 +33,20 @@
                         <userDefinedRuntimeAttribute type="number" keyPath="value">
                             <integer key="value" value="2"/>
                         </userDefinedRuntimeAttribute>
+                        <userDefinedRuntimeAttribute type="number" keyPath="trackHeight">
```

Comment:
The `trackHeight` here and the `tickWidth` below are the only things I changed. The rest of the diff is from Xcode updating from an older storyboard version.